### DISCUSSION TITLE
code example changed from PyMilvus ORM to the MilvusClient SDK

### DISCUSSION THREAD

**Discussion by youqugit** [milvus-io/milvus#33979] [Q&A and General discussion] [answered]
Why was the code example in the documentation changed from the PyMilvus ORM module to the MilvusClient SDK, and what considerations were taken into account?

**Comment by yhmo** [comment, marked as answer]
The major purpose is to make the behavior of all the SDKs to be consistent.
Python/Java/Go/Node, now all these SDKs have a MilvusClient class with similar interfaces.

**Comment by xiaofan-luan** [comment]
@czs007 can help on migrating the old API to new API.
mean time all old API will be still acceptable until 3.0

**Comment by maxshatskiy** [comment]
are there any performance differences between MilvusClient or the original ORM. for example for query?

**Reply by xiaofan-luan** [reply]
ORM will be deprecated soon and Milvus Client is the only choice
